ElectronClient/gui/KeymapConfig/KeymapConfigScreen.tsx

```diff
@@ -0,0 +1,168 @@
+import * as React from 'react';
+import { useState, useEffect } from 'react';
+
+import styles_ from './styles';
+import { ShortcutRecorder } from './ShortcutRecorder';
+import CommandService from '../../lib/services/CommandService';
+import KeymapService, { KeymapItem } from '../../lib/services/KeymapService';
+
+const { _ } = require('lib/locale.js');
+const { shim } = require('lib/shim.js');
+
+const keymapService = KeymapService.instance();
+const commandService = CommandService.instance();
+
+interface CommandLabels {
+	[commandName: string]: string
+}
+
+// Some commands aren't actually registered in CommandService
+// These hardcoded labels are used for those commands instead
+const commandLabels: CommandLabels = {
```

Comment:
Do the labels get updated when the user changes the language ?

Reply:
I wrapped this logic in a function. Now it's working even after changing the language.